### DISCUSSION TITLE
Un-Cloak Event Listener

### DISCUSSION THREAD

**Discussion by fcwheeler** [unocss/unocss#3209] [Ideas]
Hello!

I have been building with the runtime library and I needed to wait for the un-cloak attribute to be removed before I could run another script. It would be helpful to have a config property that could allow for a function execution on-finish. If this already exists, let me know!

Ryan

**Comment by chu121su12** [comment]
For me the requirement feels to specific. As alternative, use the `ready()` callback and await for the `runtime.extractAll()`

https://github.com/unocss/unocss/blob/5dcc8ceddb7b6acbb06be914b5f1bd3a444c8976/packages/runtime/src/index.ts#L49
https://github.com/unocss/unocss/blob/5dcc8ceddb7b6acbb06be914b5f1bd3a444c8976/packages/runtime/src/index.ts#L224-L232
